support-frontend/assets/pages/contributions-landing/checkoutFormIsSubmittableActions.js

```diff
@@ -116,25 +117,19 @@ function enableOrDisableForm() {
       state.page.form.userTypeFromIdentityResponse,
     );
 
-    const v2RecaptchaCheck =
-      window.guardian.recaptchaV2
-      && state.page.form.stripeCardFormData.formComplete
-      && (
-        state.common.internationalisation.countryGroupId === 'AUDCountries' ||
-        state.common.abParticipations.recaptchaPresenceTest === 'recaptchaPresent'
-      )
-      && !state.page.user.isPostDeploymentTestUser
-        ? state.page.form.v2IsLowRisk
-        : true;
-
     const formIsValid = getFormIsValid(formIsValidParameters(state));
     dispatch(setFormIsValid(formIsValid));
 
+    const auStripeRecurringNotVerified =
```

Comment:
I wouldn't prefix this with `au` because you're already passing the countryGroup into the `recaptchaEnabled` decision function so the boolean is country agnostic.

Reply:
good point